middlewares/auth.js

```diff
@@ -0,0 +1,21 @@
+const jwt = require('jsonwebtoken');
+
+const { NODE_ENV, JWT_SECRET } = process.env;
+const { DEV_SECRET } = require('../config/config');
```

Comment:
Можно лучше: использовать значения из конфигурационного файла.
_актуально если внедрить то, что я описал в комментарии про использование `process.env` внутри конфигурационного файла_
```suggestion
const { JWT_SECRET } = require('../config/config');
```